tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -1798,47 +2122,99 @@ def test_content_analyst_verify_hs_physics_questions_111275(self):
+                    ('//li[contains(@class,"user-actions-menu")]/',
+                     'a[@role="button"]')
+                )
+            )
+        ).click()
+        self.content.wait.until(
+            expect.visibility_of_element_located(
+                (By.LINK_TEXT, 'QA Content')
+            )
+        ).click()
+        self.content.find(By.CSS_SELECTOR, '#available-books').click()
+        book = self.content.wait.until(
+            expect.element_to_be_clickable(
+                (By.XPATH,
+                 '//a[contains(@class,"book")]' +
+                 '/div[@class="title-version"]' +
+                 '/span[text()="College Physics"]')
+            )
+         )
+        self.content.driver.execute_script("arguments[0].click();", book)
+        try:
+            self.content.find(By.XPATH,
+                              '//ul[@aria-labelledby=available-books"]')
+            self.content.find(By.CSS_SELECTOR, '#available-books').click()
+        except:
+            pass
+        self.content.wait.until(
+            expect.visibility_of_element_located(
+                (By.CSS_SELECTOR, ('.title'))))
+        self.content.find(By.XPATH,
+                          '//a[contains(@href,"/section/1.2")' +
+                          'and contains(@tabindex,"0")]').click()
+        self.content.wait.until(
+            expect.visibility_of_element_located(
+                (By.XPATH, '//div[contains(@class,"openstax-question")]')))
+
+        # raise NotImplementedError(inspect.currentframe().f_code.co_name)
 
         self.ps.test_updates['passed'] = True
 
     # 111277 - 033 - Content Analyst | Verify question availability
     # for College Biology with Courseware
     @pytest.mark.skipif(str(111277) not in TESTS, reason='Excluded')
     def test_content_analyst_verify_college_bio_questions_111277(self):
-        """Verify question availability for College Biology with Courseware.
-
+        """Verify question availability for College Biology Courses.
         Steps:
+        Log in as Content Analyst
+        Open the drop-down menu
+        Click "QA Contents"
+        Select "Biology For AP® Courses" in
+         the "available-books drop-down list"
+        Click a non-introductory section
 
         Expected Result:
+        Exercises are displayed.
         """
-        self.ps.test_updates['name'] = 't1.58.033' \
+        self.ps.test_updates['name'] = 't1.58.027' \
             + inspect.currentframe().f_code.co_name[4:]
         self.ps.test_updates['tags'] = [
             't1',
             't1.58',
-            't1.58.033',
-            '111277'
+            't1.58.027',
+            '111271'
         ]
         self.ps.test_updates['passed'] = False
 
         # Test steps and verification assertions
+        """College Biology for Courseware does not exist in QA Content"""
         raise NotImplementedError(inspect.currentframe().f_code.co_name)
-
-        self.ps.test_updates['passed'] = True
```

Comment:
These two lines should not be removed for a test function; SauceLabs relies on the data